src/node_file.cc

```diff
@@ -389,10 +589,19 @@ inline int SyncCall(Environment* env, Local<Value> ctx, fs_req_wrap* req_wrap,
 #define SYNC_CALL(func, path, ...)                                            \
   SYNC_DEST_CALL(func, path, nullptr, __VA_ARGS__)                            \
 
-#define SYNC_REQ req_wrap.req
+#define SYNC_REQ sync_wrap.req
 
 #define SYNC_RESULT err
 
+inline FSReqBase* GetReqWrap(Environment* env, const Local<Value>& value) {
```

Comment:
nit: we usually pass `Local`s by value instead of references because they’re effectively just pointers